test/magic_and_declared_type_test.rb

```diff
@@ -3,8 +3,14 @@
 
 class Marcel::MimeType::MagicAndDeclaredTypeTest < Marcel::TestCase
   each_content_type_fixture('name') do |file, name, content_type|
-    test "correctly returns #{content_type} for #{name} given both file and declared type" do
+    test "detects #{content_type} given magic bytes from #{name} and declared type" do
       assert_equal content_type, Marcel::MimeType.for(file, declared_type: content_type)
     end
+
+    ALIASED[content_type].each do |aliased|
+      test "detects #{content_type} given magic bytes from #{name} and aliased type #{aliased}" do
+        assert_equal content_type, Marcel::MimeType.for(file, declared_type: aliased)
+      end
+    end
```

Comment:
Test that `declared_type: alias` will resolve to the canonical type. This what'll allow browser uploads using alias types to be correctly identified.